### ISSUE TITLE
I2C Screen

### ISSUE THREAD

**Issue by nerdwa** [vpominchuk/lightmeter#9]
Is it possible to change the code from the SPI screen to I2C? I'm using the same SSD1306 but it is the I2C model.

**Comment by vpominchuk** [comment]
Yes, it's possible. Feel free to change the code and make PR, I'll review it.

**Comment by nerdwa** [comment]
Unfortunately my ability to code is pretty mediocre at best at the moment and I'd hate to bother you. Perhaps I'll eventually figure it out.
